stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/viewmodel/messages/MessagesViewModelFactory.kt

```diff
@@ -5,14 +5,15 @@ import android.content.Context
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
 import com.getstream.sdk.chat.utils.StorageHelper
+import com.getstream.sdk.chat.viewmodel.MessageComposerController
 import io.getstream.chat.android.client.ChatClient
 import io.getstream.chat.android.compose.handlers.ClipboardHandlerImpl
 import io.getstream.chat.android.compose.ui.util.StorageHelperWrapper
 import io.getstream.chat.android.offline.ChatDomain
 
 /**
  * Holds all the dependencies needed to build the ViewModels for the Messages Screen.
- * Currently builds the [MessageComposerViewModel], [MessageListViewModel] and [AttachmentsPickerViewModel].
+ * Currently builds the [MessageComposerController], [MessageListViewModel] and [AttachmentsPickerViewModel].
```

Comment:
We could probably stick with the old wording here for the docs - the controller doesn't need to be mentioned publicly, and the `MessageComposerViewModel` is still being built just like before